src/app/clusters/messaging-client/messaging-client.c

```diff
@@ -31,18 +31,19 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-/***************************************************************************//**
- * @file
- * @brief Routines for the Messaging Client plugin, which implements the client
- *        side of the Messaging cluster.
- *******************************************************************************
-   ******************************************************************************/
+/***************************************************************************/ /**
+                                                                               * @file
+                                                                               * @brief Routines for the Messaging Client plugin,
+                                                                               *which implements the client side of the Messaging
+                                                                               *cluster.
+                                                                               *******************************************************************************
+                                                                               ******************************************************************************/
 
+#include "messaging-client.h"
 #include "../../include/af.h"
 #include "../../util/common.h"
-#include "messaging-client.h"
 
-#include "app/framework/plugin/esi-management/esi-management.h"
+// #include "app/framework/plugin/esi-management/esi-management.h"
```

Comment:
I'd really prefer it if the commit structure here were as follows:

1) The commits from the PR.
2) A restyle commit.
3) A commit with whatever "make it compile" changes we are making.

Are you willing to split this commit up into those parts 2 and 3?

In either case, we should not be commenting out the include of `asi-management.h` here.  Those updates should happen when we try to actually make this file compile, which this changeset is not doing.

Reply:
it didn't actually need to be commented out,